### ISSUE TITLE
Connection and report problem

### ISSUE THREAD

**Issue by zdorcz** [anhldbk/swarm#17]
Hi,

I found 2 errors in your project.
1. When I run the example benchmark in locust, the slave is always represented as "missing".
![image](https://user-images.githubusercontent.com/51229677/77575979-9fb85000-6ed4-11ea-9427-4021c1ea1355.png)

It turns out that it is enough to change from

https://github.com/anhldbk/swarm/blob/7c1f009fb88da0cd911b3c57d59fa3f46c6b1c30/src/main/java/com/bigsonata/swarm/services/Beat.java#L56
on
` Map<String, Object> data = new HashMap<>();`

and with
https://github.com/anhldbk/swarm/blob/7c1f009fb88da0cd911b3c57d59fa3f46c6b1c30/src/main/java/com/bigsonata/swarm/services/Beat.java#L57
on
`data.put("current_cpu_usage", (int) (osBean.getProcessCpuLoad()*100));`

and restart, then everything will be connected in a way.

![image](https://user-images.githubusercontent.com/51229677/77577008-62ed5880-6ed6-11ea-8336-c30730696791.png)

2.Another problem is that no statistics are sent (no report). Swarm fact sends data to locust but locust responds in this way

![image](https://user-images.githubusercontent.com/51229677/77577289-cb3c3a00-6ed6-11ea-98d7-4914b361bebd.png)

and the statistics are still empty.

**Comment by anhldbk** [comment]
@zdorcz Thank you for using this library.


> Map<String, Object> data = new HashMap<>(); ....
I've made a commit addressing your suggestion here: https://github.com/anhldbk/swarm/commit/4f4f2103626af324a09d30676df79043e0a9f91e

> Swarm fact sends data to locust but locust responds in this way...
I can't reproduce your error. What is your Locust version?

![fact](https://user-images.githubusercontent.com/3270746/77604476-2b69c500-6f45-11ea-8f38-9ff6755e1bf4.png)

**Comment by zdorcz** [comment]
Thank you.

First I used Locusta 0.14.5 but I had problems with the slave connection (I still have the status "missing" ) so I downgraded to version 0.14.4 which I am currently working on. Every time I have these messages when I try to send a report to Locusta, I don't know yet if Locust does not accept the report format (or maybe it has some data missing), or maybe there is a problem on Swarm's side.

![image](https://user-images.githubusercontent.com/51229677/77621255-61ef1200-6f3c-11ea-9cfa-1d5c96ac37a4.png)
